OpenEdXMobile/res/layout-land/empty_screen_layout.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
use the same approach for landscape currently introduce in Portrait mode.

uses of `layout_constraintWidth_percent`, and remove `Guideline` views.